library/erlang

```diff
@@ -1,30 +1,30 @@
 # this file is generated via https://github.com/c0b/docker-erlang-otp/blob/8e61adc781b30f61cc017ffa84ec1ad8fc7d6754/generate-stackbrew-library.sh
 
-Maintainers: Mr C0B <denc716@gmail.com> (@c0b)
+Maintainers: denc716@gmail.com (@c0b)
```

Comment:
I've sent https://github.com/c0b/docker-erlang-otp/pull/19 upstream to take care of this change.